codec/codec-jackson/src/test/java/com/alipay/sofa/rpc/codec/jackson/JacksonSerializerTest.java

```diff
@@ -501,4 +511,50 @@ public void testJacksonFeature() throws UnsupportedEncodingException {
 
     }
 
+    @Test(expected = ClassCastException.class)
+    public void testGenericSofaRequestFail() throws Exception {
+        JacksonSerializer.registerGenericService(GenericService.class.getCanonicalName() + ":1.0",
```

Comment:
How to clean the state after the test?

Reply:
how about add a `clear` method with `@VisibleForTesting`